src/main/java/core/basesyntax/service/Calculator.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import java.util.List;
+import java.util.Map;
+
+public interface Calculator {
+    Map<String, Integer> calculate(List<String> notes);

```

Comment:
Delete this interface.
We do not need to create Map. And there is no any **notes** in this task. We should deal with Fruit Transactions